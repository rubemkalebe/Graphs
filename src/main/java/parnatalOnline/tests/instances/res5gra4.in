5 4
25 25 17 3
4 3 9
4 23
3
