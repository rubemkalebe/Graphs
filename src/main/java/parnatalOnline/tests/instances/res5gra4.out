13.000000
0 4, 1 2, 1 3, 3 4, 
125
0.000000;0.002740;